data/Chandraprakash-Darji.json

```
@@ -159,5 +159,9 @@
       "icon": "SiHashnode",
       "description": "People which don't watch youtube videos then can read the blogs"
     }
+  ],
+
+  "testimonials": [
+   "Deppu178"
```

You probably be using an invalid username. Acc to LinkFree's commit history, it should be `Deepu178`.

```suggestion
   "Deepu178"
```

---

Thanks for review